config/routes.rb

```diff
@@ -51,13 +51,18 @@
       end
 
       scope '/work-history' do
-        get '/length' => 'work_history#length', as: :work_history_length
-        post '/length' => 'work_history#submit_length'
+        get '/length' => 'work_history/length#length', as: :work_history_length
```

Comment:
`work_history/length#length` 🤔 
it's not very expressive, but not sure what it could be (a screenshot could help)

Reply:
`#show` is better :)

Reply:
It's this

![Screenshot 2019-10-31 at 14 48 24](https://user-images.githubusercontent.com/1650875/67957422-7f1def80-fbed-11e9-89bb-973e5dbb2f5b.png)